stdlib/stdlib.go

```diff
@@ -18,14 +18,15 @@ func AllModuleNames() []string {
 
 // GetModules returns the modules for the given names.
 // Duplicate names and invalid names are ignore.
-func GetModules(names ...string) map[string]objects.Importable {
-	modules := make(map[string]objects.Importable)
+func GetModules(names ...string) *objects.ModuleMap {
+	modules := objects.NewModuleMap()
```

Comment:
Might be better named GetModuleMap

Reply:
Also the comment for this function is better modified to reflect that as well given that modules themselves (builtin & source) have different types and the returning object is a ModuleMap